GDJS/tests/tests/spriteruntimeobject.pixiruntimegamewithassets.js

```diff
@@ -388,4 +388,53 @@ describe('gdjs.SpriteRuntimeObject (using a PixiJS RuntimeGame with assets)', fu
       expect(object.getPointY("Origin")).to.be(0);
     });
   });
+
+  it('properly computes hitboxes and point positions after the layer camera has moved', function() {
+    return gdjs.getPixiRuntimeGameWithAssets().then(runtimeGame => {
```

Comment:
nit: We can probably start using async/await in runtime tests:

```suggestion
  it('properly computes hitboxes and point positions after the layer camera has moved', async () => {
    const runtimeGame = await gdjs.getPixiRuntimeGameWithAssets();
```